Source/WebCore/platform/network/cf/CertificateInfoCFNet.cpp

```diff
@@ -98,28 +97,43 @@ std::optional<CertificateSummary> CertificateInfo::summary() const
     auto leafCertificate = checked_cf_cast<SecCertificateRef>(CFArrayGetValueAtIndex(chain.get(), 0));
     auto subjectCF = adoptCF(SecCertificateCopySubjectSummary(leafCertificate));
     summaryInfo.subject = subjectCF.get();
-#endif
 
-#if PLATFORM(MAC)
-    if (auto certificateDictionary = adoptCF(SecCertificateCopyValues(leafCertificate, nullptr, nullptr))) {
+#if HAVE(SEC_CERTIFICATE_DATE)
+    if (auto* validNotBefore = adoptCF(SecCertificateCopyNotValidBeforeDate(leafCertificate))) {
```

Comment:
This is not correct and probably doesn't even compile.  adoptCF returns a RetainPtr, which can't resolve to anything represented by "auto*"